desktop/apps/auction2/components/page/Header.js

```diff
@@ -2,57 +2,86 @@ import AddToCalendarView from 'desktop/components/add_to_calendar/react'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Registration from './Registration'
+import { connect } from 'react-redux'
 
-export default function Header (props) {
-  const { auction } = props
-  const showAddToCalendar = !(auction.isClosed() || auction.isLiveOpen())
+function Header (props) {
+  const {
+    description,
+    isAuctionPromo,
+    liveStartAt,
+    name,
+    showAddToCalendar,
+    upcomingLabel
+  } = props
```

Comment:
Once everything settles out in the responsive view updates I'll give this stuff another pass